ocfweb/docs/docs/staff/getinvolved.md

```diff
@@ -0,0 +1,227 @@
+perspective, please come to BoD meetings! Meetings always are open to all.
+
+#### Staff Meeting
+
+From 8 to 9 pm every Monday, all OCF staff (that includes you!) get together to
+discuss technology, learn from each other, and work on OCF projects. These
+meetings will be a mix of presentations, work time, and socializing. Our goal
+is to build OCF staff into a strong community of capable individuals.
+
+### Contribute
+
+We know that finding ways to contribute to the OCF can feel daunting, but we
+don’t want you to be discouraged. Don’t worry about finding the best way to
+contribute, as long as you find something that you want to work on. Here are
+some ways to get started:
+
+#### Projects
+
+We have a curated list of projects at [ocf.io/projects][projects]. Projects
+labeled “newbie” are ones we think are best fit for new staffers, but we are
+happy to provide context and mentorship on any project. In general, don’t worry
+about fully understanding the project description. If you find a something that
+interests you, respond to the thread or reach out to a veteran staffer to get
+more information.
+
+#### GitHub Issues
+
+All of our code is on GitHub (github.com/ocf). Sometimes you can find ways to
+contribute by reading open issues, but they can sometimes be hard to understand
+since they’re short on context. Some issues are also marked as “good first
+issue.” Here’s a list of our most active repos, with some examples of things we
+can do with them:
+
+* [puppet][puppet] ([issues][puppet/issues]) - contains Puppet modules, which
+  controls the software and configuration on all of our servers.
+   * Install software packages on the desktops
+     ([ocf_desktop/manifests/packages.pp][puppet-desktop-packages])
+   * Configure Firefox to show the "are you sure you want to quit" dialog
+     ([PR #373][puppet-373])
+   * Allow operations staff to refund pages ([PR #368][puppet-368])
+   * Monitor our mirrors to see if they’re out of date
+     ([healthcheck script][mirror-healthcheck],
+     [configuring server to call the healtcheck
+     script][mirror-healthcheck-puppet],
+     [Prometheus rules for alerting][prometheus-mirror])
+   * Monitor printers to see if they’re out of paper, toner, or jammed
+     ([printer.rules.yaml][prometheus-printer])
+   * Send a desktop notification if a print jobs fails ([PR #321][puppet-321])
+* [ocfweb][ocfweb] ([issues][ocfweb/issues]) - the source code for our website,
+  www.ocf.berkeley.edu. Written in Python using the Django framework.
+   * Hosts our documentation for users and staffers (including
+     [this page][getinvolved-src]!) ([docs][docs-src])
+   * JSON APIs for lab hours ([api/hours.py][api-hours])
+   * [Account creation][account-register], [password resets][account-chpass],
+     [others][account]
+   * Lab usage statistics ([stats/][ocfweb])
+   * Form for requesting a virtual host ([lab_reservations/][lab_reservations])
```

Comment:
wat

I think you forgot to update the text, also you probably want `[ocfweb-lab_reservations]`